imageio/imageio-psd/src/main/java/com/twelvemonkeys/imageio/plugins/psd/PSDLayerInfo.java

```diff
@@ -191,4 +232,22 @@ public String toString() {
         builder.append("]");
         return builder.toString();
     }
+
+    enum SectionDividerSetting {
+        LAYER(0), OPEN_FOLDER(1), CLOSED_FOLDER(2), BOUNDING_SECTION_DIVIDER(3);
+        SectionDividerSetting(int value) { this.value = value;}
+
+        private final int value;
```

Comment:
Why do we need the `value`? It seems to always equal the enum `ordinal`.

Reply:
ordinal can used as value of SectionDividerSetting.

But I prefer use value when enum derived from the value.

Because, a position of enum can be changed then it can't stand the value.

So I prefer using value.

If the code style is use ordinal then value, i will follow the rule  :)